LuaMenu/widgets/chobby/utilities/buttons.lua

```diff
@@ -3,7 +3,7 @@ ButtonUtilities = ButtonUtilities or {}
 
 local function GetFont(size)
 	return {
-		size = size + 2,
```

Comment:
Why? This changes the alignment of everything and I doubt everything this change affects has been tested.